### ISSUE TITLE
Documentation of ExctractionTurbineCHP

### ISSUE THREAD

**Issue by jakob-wo** [oemof/oemof#526]
Hi,
I was just trying to copy&paste and run the example for the ExtractionTurbineCHP given in the documentation (oemof-solph --> solph components --> ExtractionTurbineCHP):
```python
solph.ExtractionTurbineCHP(
    label='variable_chp_gas',
    inputs={b_gas: solph.Flow(nominal_value=10e10)},
    outputs={b_el: solph.Flow(), b_th: solph.Flow()},
    conversion_factors={b_el: 0.3, b_th: 0.5},
    conversion_factor_single_flow={b_el: 0.5})
```
Comparing this declaration with the component's docstring it seems like the argument `conversion_factor_single_flow` was renamed to `conversion_factor_full_condensation`. Is that correct?

**Comment by uvchik** [comment]
You are right. This is a bug in the documentation. May you open a PR and fix it?

**Comment by jakob-wo** [comment]
Sure! :)

Is there a way for me to display the updated rst-files in the final "layout" as they will appear on the readthedocs-docu-page later? I did not figure out how to do it.

**Comment by jnnr** [comment]
Can we close this issue?
